chromium_src/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_edit_dialog.ts

```diff
@@ -3,11 +3,13 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-import { RegisterStyleOverride } from 'chrome://resources/brave/polymer_overriding.js'
-import { html } from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js'
+import { RegisterStyleOverride } from '//resources/brave/polymer_overriding.js'
+import { html } from '//resources/polymer/v3_0/polymer/polymer_bundled.min.js'
 
 RegisterStyleOverride('power-bookmarks-edit-dialog', html`<style>
   :host {
     --leo-button-padding: 4px;
   }
 </style>`)
+
+export * from './power_bookmarks_edit_dialog-chromium.js'
```

Comment:
I feel like all imports should be at the top (export * is still an import). This makes it seem like the file will be imported after our code has run. Unless there's something I'm not thinking about from when this will be bundled?

Reply:
So I've moved these to just after the imports - even though our bundler does lift this to happen first it actually doesn't need to (Vite on Codesandbox runs the imports in the written order) but I agree its probably better to have our flow match the code.